src/main/resources/public/ts/models/ticket.model.ts

```diff
@@ -0,0 +1,118 @@
+import {Attachment} from "./Attachment";
+
+export interface ITicketResponse {
+    id: number;
+    subject: string;
+    description: string;
+    category: string;
+    school_id: string;
+    status: number;
+    newComment: string;
+    attachments: [];
+}
+
+export interface ITicketPayload{
+}
```

Comment:
si je me souviens, ton formulaire de misse a jour de ticket passe un subject: string; en paramètre, non ?
```suggestion
export interface ITicketPayload{
    subject: string;
}
```